keyboards/ryanbaekr/rb1/info.json

```
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "rb1",
+    "manufacturer": "ryanbaekr",
+    "url": "",
+    "maintainer": "ryanbaekr",
+    "usb": {
+        "vid": "0x7262",
+        "pid": "0x0001",
+        "device_version": "0.0.3"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"x":0, "y":0}
```

```suggestion
                {"matrix": [0, 0], "x":0, "y":0}
```
rb1.[ch] can be deleted with this :)